### PR TITLE
chore(docs): release note 2020.10

### LINKED ISSUE #4670: release notes 2020.10
**Describe the issue. What is the expected and unexpected behavior?**
release notes 2020.10

### PR DESCRIPTION
**Pull request by tlabaj**
**What**: Closes #4670

### CODE REVIEW

File: packages/react-docs/RELEASE-NOTES.md [hunk 1]
```diff
@@ -4,6 +4,96 @@ section: overview
...
+- [@patternfly/react-styles@4.7.0](https://www.npmjs.com/package/@patternfly/react-styles/v/4.7.0)
+- [@patternfly/react-table@4.15.2](https://www.npmjs.com/package/@patternfly/react-table/v/4.15.2)
+- [@patternfly/react-tokens@4.9.1](https://www.npmjs.com/package/@patternfly/react-tokens/v/4.9.1)
+- [@patternfly/react-topology@4.4.60](https://www.npmjs.com/package/@patternfly/react-topology/v/4.4.60)
+- [@patternfly/react-virtualized-extension@4.5.48](https://www.npmjs.com/package/@patternfly/react-virtualized-extension/v/4.5.48)
+
+### Components
+- **Button:**
+  -  Added cta/large modifier ([#4596](https://github.com/patternfly/patternfly-react/pull/4596))
+- **ChipGroup:**
```
Comment (redallen) [inline packages/react-docs/RELEASE-NOTES.md:22]: Chip group

File: packages/react-docs/RELEASE-NOTES.md [hunk 2]
```diff
@@ -4,6 +4,96 @@ section: overview
...
+  - Added type button to overflow chip button ([#4608](https://github.com/patternfly/patternfly-react/pull/4608))
+  - Changed TimesIcon to TimesCircleIcon ([#4550](https://github.com/patternfly/patternfly-react/pull/4550))
+- **DescriptionList:**
```
Comment (redallen) [inline packages/react-docs/RELEASE-NOTES.md:25]: Description list

File: packages/react-docs/RELEASE-NOTES.md [hunk 3]
```diff
@@ -4,6 +4,96 @@ section: overview
...
+  - Fix accessibility issues and add accessibility test cases ([#4466](https://github.com/patternfly/patternfly-react/pull/4466))
+- **Overflow menu:**
+  - Updated to allow0 passing dropdown item props to overflow menu dropdown item ([#4536](https://github.com/patternfly/patternfly-react/pull/4536))
+- **Page** 
+  - Fixed breakpoint typo in PageHeaderToolsItem: ([#4634](https://github.com/patternfly/patternfly-react/pull/4634))
+  - Used the correct type for the page context ([#4576](https://github.com/patternfly/patternfly-react/pull/4576))
+- **Popover:** 
+  - Added reference prop ([#4546](https://github.com/patternfly/patternfly-react/pull/4546))
+  - Updated to propagate clicks ([#4585](https://github.com/patternfly/patternfly-react/pull/4585))
+- **SearchInput:**
```
Comment (redallen) [inline packages/react-docs/RELEASE-NOTES.md:51]: Search input

File: packages/react-docs/RELEASE-NOTES.md [hunk 4]
```diff
@@ -4,6 +4,96 @@ section: overview
...
+  - Changed spelling of visiblity prop to visibility([#4560](https://github.com/patternfly/patternfly-react/pull/4560))
+  - Fixed so delete chip does not expects categoryKey as string ([#4553](https://github.com/patternfly/patternfly-react/pull/4553))
+- **Tooltip:** 
+  - Added reference prop ([#4546](https://github.com/patternfly/patternfly-react/pull/4546))
+- **Wizard:** 
+  - Made wizard nav items more easily discoverable ([#4425](https://github.com/patternfly/patternfly-react/pull/4425))
+
+### Charts
+  - Updated chart imports ([#4645](https://github.com/patternfly/patternfly-react/pull/4645))
+   - Update Victory packages to support OUIA props ([#4633](https://github.com/patternfly/patternfly-react/pull/4633))
```
Comment (redallen) [inline packages/react-docs/RELEASE-NOTES.md:69]: 1 less space

File: packages/react-docs/RELEASE-NOTES.md [hunk 5]
```diff
@@ -4,6 +4,96 @@ section: overview
...
+  - Replace classnames with css function ([#4582](https://github.com/patternfly/patternfly-react/pull/4582))
+- **Demos:** 
+  - Fixed demo headers and imports ([#4607](https://github.com/patternfly/patternfly-react/pull/4607))
+  - Added notification drawer demo ([#4640](https://github.com/patternfly/patternfly-react/pull/4640))
+- **Docs:**
+  - Updated to use h3s for titles ([#4570](https://github.com/patternfly/patternfly-react/pull/4570))
+  - Added missing imports to React doc ([#4627](https://github.com/patternfly/patternfly-react/pull/4627))
+  - Moved non-patternfly deps to devDeps in React docs ([#4584](https://github.com/patternfly/patternfly-react/pull/4584))
+  - Fixed title syntax in react docs ([#4628](https://github.com/patternfly/patternfly-react/pull/4628))
+  - Fix typos in upgrade guide ([#4601](https://github.com/patternfly/patternfly-react/pull/4601))
```
Comment (redallen) [inline packages/react-docs/RELEASE-NOTES.md:95]: Fixes

File: packages/react-docs/RELEASE-NOTES.md [hunk 6]
```diff
@@ -4,6 +4,96 @@ section: overview
...
+
+### Topology
+  - Moved @types to the dependencies list ([#4569](https://github.com/patternfly/patternfly-react/pull/4569))
+  - Re-parented nodes with invalid parents to graph ([#4629](https://github.com/patternfly/patternfly-react/pull/4629))
+
+### Other
+- **Chore:**
+  - Fixed sentence case in 2020.09 notes ([#4571](https://github.com/patternfly/patternfly-react/pull/4571))
+  - Bumped lodash to 4.17.19 ([#4580](https://github.com/patternfly/patternfly-react/pull/4580))
+  - Replace classnames with css function ([#4582](https://github.com/patternfly/patternfly-react/pull/4582))
```
Comment (redallen) [inline packages/react-docs/RELEASE-NOTES.md:86]: Replaced

File: packages/react-docs/RELEASE-NOTES.md (hunk 4 above)
Comment (redallen) [reply packages/react-docs/RELEASE-NOTES.md:69]: Also Updated

File: packages/react-docs/RELEASE-NOTES.md [hunk 7]
```diff
@@ -4,6 +4,96 @@ section: overview
...
+- **Hint:**
+  - Added hint components ([#4595](https://github.com/patternfly/patternfly-react/pull/4595))
+- **Modal:** 
+  - Updated to ensure getElement does not return null ([#4544](https://github.com/patternfly/patternfly-react/pull/4544))
+- **Nav expandable:**
+  - Converted nav expandable to button ([#4512](https://github.com/patternfly/patternfly-react/pull/4512))
+- **Notification badge:**
+  - Added needs attention state and count ([#4516](https://github.com/patternfly/patternfly-react/pull/4516))
+- **Notification drawer:**
+  - Fix accessibility issues and add accessibility test cases ([#4466](https://github.com/patternfly/patternfly-react/pull/4466))
```
Comment (redallen) [inline packages/react-docs/RELEASE-NOTES.md:42]: Fixed

File: packages/react-docs/RELEASE-NOTES.md [hunk 8]
```diff
@@ -4,6 +4,96 @@ section: overview
...
+- [@patternfly/react-inline-edit-extension@4.5.58](https://www.npmjs.com/package/@patternfly/react-inline-edit-extension/v/4.5.58)
+- [@patternfly/react-styles@4.7.0](https://www.npmjs.com/package/@patternfly/react-styles/v/4.7.0)
+- [@patternfly/react-table@4.15.2](https://www.npmjs.com/package/@patternfly/react-table/v/4.15.2)
+- [@patternfly/react-tokens@4.9.1](https://www.npmjs.com/package/@patternfly/react-tokens/v/4.9.1)
+- [@patternfly/react-topology@4.4.60](https://www.npmjs.com/package/@patternfly/react-topology/v/4.4.60)
+- [@patternfly/react-virtualized-extension@4.5.48](https://www.npmjs.com/package/@patternfly/react-virtualized-extension/v/4.5.48)
+
+### Components
+- **Button:**
+  -  Added cta/large modifier ([#4596](https://github.com/patternfly/patternfly-react/pull/4596))
```
Comment (jschuler) [inline packages/react-docs/RELEASE-NOTES.md:21]: Spell out call-to-action ?
```suggestion
  -  Added isLarge modifier prop (call to action) ([#4596](https://github.com/patternfly/patternfly-react/pull/4596))
```

File: packages/react-docs/RELEASE-NOTES.md [hunk 9]
```diff
@@ -4,6 +4,96 @@ section: overview
...
+- **ChipGroup:**
+  - Added type button to overflow chip button ([#4608](https://github.com/patternfly/patternfly-react/pull/4608))
+  - Changed TimesIcon to TimesCircleIcon ([#4550](https://github.com/patternfly/patternfly-react/pull/4550))
```
Comment (jschuler) [inline packages/react-docs/RELEASE-NOTES.md:24]:
```suggestion
  - Changed TimesIcon to TimesCircleIcon in Chips ([#4550](https://github.com/patternfly/patternfly-react/pull/4550))
```

**Pull request merged**